toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -315,7 +315,7 @@ func addOrUpdateUser(user imagecustomizerapi.User, baseConfigPath string, imageC
 		}
 
 		// Add the user.
-		err = userutils.AddUser(user.Name, hashedPassword, uidStr, imageChroot)
+		err = userutils.AddUser(user.Name, user.HomeDirectory, user.PrimaryGroup, hashedPassword, uidStr, imageChroot.(*safechroot.Chroot))
```

Comment:
The `ConfigureUserGroupMembership` call below should be skipped if the user is added.

Reply:
We need to configure secondary groups still, even if the primary group is ok. I'm wondering if the extra code complexity is worth it here to decide if we need to pass "" or `user.PrimaryGroup` in...

Reply:
ok, added conditional `var primaryGroup string`.